subxt/src/config/extrinsic_params.rs

```diff
@@ -57,13 +57,20 @@ pub trait ExtrinsicParams<Hash>: Debug + 'static {
 #[derive(Derivative)]
 #[derivative(Debug(bound = "Tip: Debug"))]
 pub struct BaseExtrinsicParams<T: Config, Tip: Debug> {
-    era: Era,
-    nonce: u64,
-    tip: Tip,
-    spec_version: u32,
-    transaction_version: u32,
-    genesis_hash: T::Hash,
-    mortality_checkpoint: T::Hash,
+    /// era
+    pub era: Era,
+    /// nonce (account nonce sent along an extrinsic, such that no extrinsic is submitted twice)
+    pub nonce: u64,
+    /// tip
+    pub tip: Tip,
+    /// spec version
+    pub spec_version: u32,
+    /// transaction version
```

Comment:
Nit: could use a bit more details for these docs since we'd expose them to the end-user